Library/Homebrew/test/support/helper/formula.rb

```diff
@@ -6,6 +6,7 @@ module Formula
       def formula(name = "formula_name", path: Formulary.core_path(name), spec: :stable, alias_path: nil, &block)
         Class.new(::Formula, &block).new(name, path, spec, alias_path: alias_path)
       end
+      alias_method stub_formula formula
```

Comment:
The method name `formula` clashes with…
https://github.com/Homebrew/brew/blob/0aab25bdb3bfa96d8eb14a38001ff6a24587bcdd/Library/Homebrew/test/formulary_spec.rb#L139